web_external/Datasets/UploadDatasetView.js

```diff
@@ -125,49 +166,18 @@ isic.views.UploadDatasetView = isic.View.extend({
         this.uploadWidget.uploadNextFile();
     },
 
-    submitDataset: function () {
+    submitDataset: function (zipFileId) {
```

Comment:
To be clear, if `zipFileId === null`, we make a REST request anyway, and let the server be the one to determine it's an error? I don't actually see anything wrong with this, but maybe make that clear in a code comment.